openshift/Makefile

```diff
@@ -1,14 +1,17 @@
 NIGHTLY_YAML=https://raw.githubusercontent.com/openshift/tektoncd-pipeline/release-next/openshift/release/tektoncd-pipeline-nightly.yaml
-STABLE_VERSION=$(shell curl -s https://api.github.com/repos/tektoncd/pipeline/releases | python -c "import sys, json;x=json.load(sys.stdin);print(x[0]['tag_name'])")
+STABLE_PIPELINE_VERSION=$(shell curl -s https://api.github.com/repos/tektoncd/pipeline/releases | python -c "import sys, json;x=json.load(sys.stdin);print(x[0]['tag_name'])")
+STABLE_RELEASE_YAML=https://raw.githubusercontent.com/openshift/tektoncd-pipeline/release-$(STABLE_PIPELINE_VERSION)/openshift/release/tektoncd-pipeline-$(STABLE_PIPELINE_VERSION).yaml
```

Comment:
What if a release happen upstream (0.9.0) and we still haven't done the downstream one ? this will result in a 404 right ?

Reply:
yep indeed, but it fails we need to fix it (release often), and that's only a fallback anyway if nightly has failed, i can have static version in there, but that's one more step to update/take care of when doing the downstream pipeline release :\ (which me no like)

Reply:
Alright, let's do that in a follow-up :stuck_out_tongue:

Reply:
you want to do version pinning? or you want to use upstream release.yaml?